README.md

```diff
@@ -16,17 +16,20 @@ On a mac, run this:
 
 `zsh webserver-darwin.sh`
 
-On arch linux run this:
+On Arch linux, the best linux, run this:
+`bash webserver-archlinux.sh`
+
+on linux run this:
 
 `bash webserver-linux.sh`
 
 On debian or ubuntu linux run this:
 
 `bash webserver-ubuntu.sh`
 
-On Windows open powershell as administrator and run:
-`Set-ExecutionPolicy Bypass` 
-`./powershell-test.ps1`
+On Windows do the following:
+  - open powershell as administrator set the system's security policy: `Set-ExecutionPolicy RemoteSigned` This allows scripts to be run locally while requiring scripts downloaded from the internet to be digitally signed.
```

Comment:
What if we just combine this with the command below? Just put it on same command.

Reply:
I can make that happen